src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/53.diff.sql

```diff
@@ -0,0 +1,1223 @@
+            SingleValue         DECIMAL (36, 18) NULL,
+            LowValue            DECIMAL (36, 18) NULL,
+            HighValue           DECIMAL (36, 18) NULL UNIQUE (ResourceTypeId, ResourceSurrogateId, SearchParamId, SingleValue, LowValue, HighValue));
+
+    IF NOT EXISTS (SELECT * FROM sys.types WHERE name = 'Tmp_QuantitySearchParamList')
+        CREATE TYPE dbo.Tmp_QuantitySearchParamList AS TABLE (
+            ResourceTypeId      SMALLINT        NOT NULL,
+            ResourceSurrogateId BIGINT          NOT NULL,
+            SearchParamId       SMALLINT        NOT NULL,
+            SystemId            INT             NULL,
+            QuantityCodeId      INT             NULL,
+            SingleValue         DECIMAL (36, 18) NULL,
+            LowValue            DECIMAL (36, 18) NULL,
+            HighValue           DECIMAL (36, 18) NULL UNIQUE (ResourceTypeId, ResourceSurrogateId, SearchParamId, SystemId, QuantityCodeId, SingleValue, LowValue, HighValue));
+
+
+    IF NOT EXISTS (SELECT * FROM sys.types WHERE name = 'Tmp_TokenNumberNumberCompositeSearchParamList')
+        CREATE TYPE dbo.Tmp_TokenNumberNumberCompositeSearchParamList AS TABLE (
+            ResourceTypeId      SMALLINT        NOT NULL,
+            ResourceSurrogateId BIGINT          NOT NULL,
+            SearchParamId       SMALLINT        NOT NULL,
+            SystemId1           INT             NULL,
+            Code1               VARCHAR (256)   COLLATE Latin1_General_100_CS_AS NOT NULL,
+            CodeOverflow1       VARCHAR (MAX)   COLLATE Latin1_General_100_CS_AS NULL,
+            SingleValue2        DECIMAL (36, 18) NULL,
+            LowValue2           DECIMAL (36, 18) NULL,
+            HighValue2          DECIMAL (36, 18) NULL,
+            SingleValue3        DECIMAL (36, 18) NULL,
+            LowValue3           DECIMAL (36, 18) NULL,
+            HighValue3          DECIMAL (36, 18) NULL,
+            HasRange            BIT             NOT NULL);
+
+
+    IF NOT EXISTS (SELECT * FROM sys.types WHERE name = 'Tmp_TokenQuantityCompositeSearchParamList')
+        CREATE TYPE dbo.Tmp_TokenQuantityCompositeSearchParamList AS TABLE (
+            ResourceTypeId      SMALLINT        NOT NULL,
+            ResourceSurrogateId BIGINT          NOT NULL,
+            SearchParamId       SMALLINT        NOT NULL,
+            SystemId1           INT             NULL,
+            Code1               VARCHAR (256)   COLLATE Latin1_General_100_CS_AS NOT NULL,
+            CodeOverflow1       VARCHAR (MAX)   COLLATE Latin1_General_100_CS_AS NULL,
+            SystemId2           INT             NULL,
+            QuantityCodeId2     INT             NULL,
+            SingleValue2        DECIMAL (36, 18) NULL,
+            LowValue2           DECIMAL (36, 18) NULL,
+            HighValue2          DECIMAL (36, 18) NULL);
+
+    SET @logTime = getUTCdate()
+    EXECUTE dbo.LogEvent @Process='Creating temp tables',@Status='End',@Start=@logTime
```

Comment:
Setting start date write before LogEvent does not make sense as internally it is used only to calculate duration of operation, which in this case will be always zero.

Reply:
Removed SET @logTime = getUTCdate() line adn also @Start from LogEvent call